README.md

```diff
@@ -38,16 +44,21 @@ Since decorators are still in proposal state, you need to add the following plug
 $ yarn add -D babel-plugin-transform-typescript-metadata @babel/plugin-proposal-decorators babel-plugin-parameter-decorator
 ```
 
-Make sure to add the following lines to the `plugins` section in your babel configuration file:
-```json
-"babel-plugin-transform-typescript-metadata",
-["@babel/plugin-proposal-decorators", { "legacy": true }],
-"babel-plugin-parameter-decorator",
+Make sure to add the following lines to the start of the `plugins` section in your babel configuration file:
```

Comment:
Putting these plugins to the top didn't solve the conflict. So we better encourage devs to add these to a babel config file under `pages/api` folder.